other/amo.md

```diff
@@ -0,0 +1,35 @@
+# ⚙️ Angle AMOs
+
+## What are AMOs
+
+AMO stands for algorithmic market operations. AMOs are operations performed by contracts to mint or burn stablecoins, provided that this new supply is properly backed in one way or another, and doesn't affect the peg of the stablecoin. 
+
+### AMO on lending markets
+
+One type of such operations could be an AMO on lending market. In this case, a contract mints an amount of stablecoins, in our case an agToken, and deposits them on a lending market such as Aave or Euler. Then, these stablecoins can be borrowed by people having a sufficient amount collateral in the lending protocol. Once borrowed, the minted stablecoins are finally on the open market. At this point, they are backed by the collateral deposited in the lending protocol by the borrowers. As such, the agTokens originally minted by the protocol contract and deposited in the lending market are over-collateralized when being released in the market. 
+
+This mechanism has several effects, among which an expansion of the agTokens supply and a lowering of the deposit and borrow APYs in the affected lending market. 
+
+### Pros and cons
+
+The main benefit of these operations is that they allow users of the lending markets to have access to agTokens at a lower borrowing cost, effectively widening the potential user base for Angle and agTokens. However, the direct consequence is that agTokens depositors earn a lower yield on their deposits. 
+
+Another benefit is that it allows the protocol to earn a yield on these deposited tokens, to further fund its operations. 
+
+{% hint style="succes" %} 
+Borrowed stablecoins are fully backed by the borrowers' deposits in the lending market. 
+{% endhint %}
+
+## Angle AMOs on lending markets
+
+As you can see, AMOs can be a great way to help expand agTokens. Doing these operations on robust and battle-tested lending markets also ensures that the stablecoins ending up in the open market are backed by collateral. 
```

Comment:
Try to avoid As you can see in the docs, sounds a bit familiar for something which needs to be impersonal (contrary to an article). Here we can just keep the paragraph on approved AMOs